fastlane/lanes/build.rb

```diff
@@ -385,6 +409,13 @@ def build_and_upload_prototype_build(scheme:, output_app_name:, appcenter_app_na
       dsym_path: lane_context[SharedValues::DSYM_OUTPUT_PATH]
     )
 
+    upload_gutenberg_sourcemaps(
+      sentry_project_slug: sentry_project_slug,
+      release_version: release_version_current,
+      build_version: build_number,
+      app_identifier: app_identifier
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/wordpress-mobile/WordPress-iOS/blob/a1a692f09379dc1e081781d78ae488238eb956a6/fastlane/lanes/build.rb#L416"></span>Style/HashSyntax: Omit the hash value.</td>
    </tr>
  </tbody>
</table>